lab_3_ann_retriever/main.py

```diff
@@ -49,6 +51,12 @@ def calculate_distance(query_vector: Vector, document_vector: Vector) -> float |
 
     In case of corrupt input arguments, None is returned.
     """
+    dist = 0
+    i = 0
+    while i < len(query_vector):
```

Comment:
I prefer `for` loop over `zip` of `query_vector` and `document_vector`

Reply:
done